README.md

```diff
@@ -185,6 +185,85 @@ the same directory. This can be helpful if you need to create factories that are
 used for different repos, your factory module is getting too big, or if you have
 different ways of saving the record for different types of factories.
 
+### How to split your factories into different files
+
+We will assume that you use factories only in test environment. If you use them in all environments, just adjust directory structure.
+
+> Start by creating main factory module in `test/support/factory.ex` and name it `MyApp.Factory`.
+
+```elixir
+# test/support/factory.ex
+defmodule MyApp.Factory do
+  use ExMachina.Ecto, repo: MyApp.Repo
+
+  use MyApp.ArticleFactory
+end
+```
+
+`MyApp.ArticleFactory` will be the first factory we will create. Create separate directory for factories, like `test/factories`. Here is how to create a factory:
```

Comment:
How about:

"Main factory includes `ArticleFactory`, let's create it next. It might be useful to keep the factories in separate directory, like `test/factories`:"